changelog/unreleased/4438

```diff
@@ -0,0 +1,10 @@
+Enhancement: Hardware keyboard support
+
+Navigation via hardware keyboard has been improved so that now focus order has a logical path, every element
+is reachable and there are no traps. These improvements have been applied in main file list, spaces list,
+drawer menu, share view and image preview.
+
+https://github.com/owncloud/android/pull/4438
```

Comment:
I placed it in the first place because this is a PR linked to 3 different issues, there is not a main issue and some others that are also completed as a collateral effect, but 3 "main" issues. What all of them have in common is the PR